packit_service/service/templates/build_info.html

```diff
@@ -11,7 +11,10 @@ <h1>
   <a
     href="{{ url_for('builds.get_srpm_build_logs_by_id', id_=build.srpm_build_id) }}"
   >
-    SRPM build
+    SRPM build {% if build.srpm_build.url %} (<a
+      href="{{ build.srpm_build.url }}"
+      >Download SRPM</a
+    >){% endif %}
```

Comment:
What about not nesting the links?

```suggestion
    SRPM build</a>
    {% if build.srpm_build.url %} (<a
      href="{{ build.srpm_build.url }}"
      Download SRPM){% endif %}
```

(Note that I can't suggest an edit of the existing line so we need to add a right parenthesis after the `</a>` on the next line...)